loadNeustarReputationIntoES.js

```diff
@@ -303,22 +320,27 @@ function deleteUnusedIndexes(deleteAll) {
                         var somethingDeleted = false;
                         for (i = 0; i < arrayOfIndexes.length; i++) {
                             if (arrayOfIndexes[i] != currentMonolithIndex) {
-                                somethingDeleted = true;
-                                deleteIndex(arrayOfIndexes[i]);
-                                if (arrayOfIndexes[i] === loadedIndex) {
-                                    removeLoadedIndexFromScratchSpace();
-                                }
-                                if (arrayOfIndexes[i] === pausedIndex) {
-                                    removePausedIndexFromScratchSpace();
-                                }
                                 if (arrayOfIndexes[i] === lastUsedIndex) {
-                                    removeLastUsedIndexFromScratchSpace();
+                                    winston.log('verbose', "Last used index " + lastUsedIndex + " must not be deleted.");
+                                    continue;
```

Comment:
This is where I avoid deleting the index if it was the last used one by monolith.